sssd_test_framework/utils/tools.py

```diff
@@ -586,6 +586,40 @@ def teardown(self):
 
         super().teardown()
 
+    def wait_for(self, condition: str, body: str = "", timeout: int = 60) -> SSHProcessResult:
+        """
+        Wait at maximum ``timeout`` seconds until the ``condition`` is true. Execute ``body`` after each attempt.
+        The condition is a bash expression, usually it is a single bash command that must succeed before a test
+        can continue.
+
+        .. note::
+            Internally, this expands to ``timeout {time}s bash -c 'until {condition}; do : {body}; done'``.
```

Comment:
You need to add a blank line before the paragraph.

Reply:
Okay and why? Docs seems to look okay

Reply:
That is a surprise to me :-) RST syntax is:

```
.. directive:: arguments
    options

    content
```